docs/libraries/go.md

```diff
@@ -67,14 +67,16 @@ func main() {
 }
 ```
 To build, create a directory for the main package, initialize it,
-copy the above code into main.go, fetch the quired zed packages.
+copy the above code into `main.go`, fetch the required Zed packages.
 ```
 mkdir example
 cd example
-cat > main.go < [paste from above]
+go mod init example
+cat - > main.go
+# [paste from above]
 go get github.com/brimdata/zed
-go get github.com/brimdata/zson
-go get github.com/brimdata/zio/zsonion
+go get github.com/brimdata/zed/zson
+go get github.com/brimdata/zed/zio/zsonio
```

Comment:
I fixed this because it contained a typo, though strictly speaking it doesn't appear this import is not actually required. Happy to remove once y'all Go developers acknowledge that.

Reply:
@nwt: I was able to dump all three `go get` command lines by doing a single `go mod tidy` instead. Please confirm this is the appropriate way to handle this from a Go perspective.